ui/app/common/attributeTypes/directives/attributeTypes.js

```diff
@@ -56,7 +56,10 @@ angular.module('bahmni.common.attributeTypes', []).directive('attributeTypes', [
                     return answer.conceptId === attributeValueConceptType.conceptUuid;
                 });
                 attributeValueConceptType.value = concept && concept.fullySpecifiedName;
-                $rootScope.typeOfRegistrationSelected = attributeValueConceptType.value;
+                if (attributeValueConceptType.value === 'NEW_PATIENT' || attributeValueConceptType.value === 'TRANSFERRED_PATIENT' || attributeValueConceptType.value === undefined) {
```

Comment:
@simaomazive when I test this bug, I am unable to replicate locally. Without this change the FacilityCode is not affected when I select other drop downs on form.

is there a way for me to better replicate the issue?